Dockerfile

```diff
@@ -47,7 +47,7 @@ ENV CC=gcc CXX=g++ CFLAGS="-U__sun__" RUSTFLAGS="-C target-feature=-crt-static"
 RUN rustup toolchain install nightly
 ENV CARGO_UNSTABLE_SPARSE_REGISTRY=true
 
-RUN cargo +nightly build --release --no-default-features --features=ecc608,tpm
+RUN cargo +nightly build --release --features=tpm
```

Comment:
why not also support ecc608 in one image? or is that done by default + this features list?

Reply:
I believe you said it does

Reply:
Yeah. Default enables validator and ecc

Reply:
ok, may want to take it out of draft?